cmdx.py

```diff
@@ -3753,6 +3754,11 @@ def _python_to_plug(value, plug):
         for index, value in enumerate(value):
             _python_to_plug(value, plug[index])
 
+    elif isinstance(value, om.MColor):
```

Comment:
Note to future readers, I initially tried the same approach as with MVector, but for whatever reason I kept getting these black-box Maya errors about it. Unrolling that loop like this however works just fine.